src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.CinemaHallRequestDto;
+import mate.academy.spring.dto.CinemaHallResponseDto;
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.mapper.CinemaHallDtoMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private final CinemaHallService cinemaHallService;
+    private final CinemaHallDtoMapper dtoMapper;
+
+    public CinemaHallController(CinemaHallService cinemaHallService,
+                                CinemaHallDtoMapper dtoMapper) {
+        this.cinemaHallService = cinemaHallService;
+        this.dtoMapper = dtoMapper;
+    }
+
+    @GetMapping("")
+    public List<CinemaHallResponseDto> getAllCinemaHalls() {
+        return cinemaHallService.getAll().stream()
+                .map(dtoMapper::parse)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping("")
+    public CinemaHallResponseDto add(CinemaHallRequestDto requestDto) {
```

Comment:
also check your code for it
```suggestion
    @GetMapping
    public List<CinemaHallResponseDto> getAllCinemaHalls() {
        return cinemaHallService.getAll().stream()
                .map(dtoMapper::parse)
                .collect(Collectors.toList());
    }

    @PostMapping
    public CinemaHallResponseDto add(CinemaHallRequestDto requestDto) {
```